tasks/Estimating Geological Metrics.md

```diff
@@ -0,0 +1,69 @@
+# Summary
+
+Participants were organized into groups of four and performed a series of tasks in a series of rounds with old and new group members. This task was an intellective task. participants were asked to correctly answer a series of questions without looking up the solutions online, instead debating with their group members on what the answers to the questions could be. At the end teams decided on the correct answer for each prompt and one person submitted a list on behalf of the team 
+
+# References
+> Intellective Tasks: Anita Williams Woolley, Christopher F Chabris, Alex Pentland, Nada Hashmi, and Thomas W Malone. 2010. Evidence
+for a collective intelligence factor in the performance of human groups. science 330, 6004 (2010), 686–688.
+
```

Comment:
I'm not sure if you looked at this, but it was in the task excel sheet. https://hci.stanford.edu/publications/2019/fracture/fracture-cscw2019.pdf